packages/compass-home/src/components/home.tsx

```diff
@@ -143,6 +152,25 @@ function notifyMainProcessOfDisconnect() {
   void ipcRenderer?.call('compass:disconnected');
 }
 
+export const CompassConnectionProviderContext: React.FunctionComponent<{}> = ({
+  children,
+}) => {
+  const storage = useContext(ConnectionStorageContext);
+  if (!storage) {
+    throw new Error(
+      'Could not find current ConnectionStorage. Did you forget to setup the ConnectionStorageContext?'
+    );
+  }
```

Comment:
Similarly here, it might be nice to just export a function that does what the 6 lines here do from the connection-storage package

Reply:
Yeah, I'll need help with that (I'll ping you on Slack). My idea was to have it in connection-storage, but I had a few problems setting up the package because it doesn't support TSX.